.gitignore

```diff
@@ -0,0 +1,19 @@
+# Maven projects
+/cgmes-validation/target/
+/distribution-validation/target/
```

Comment:
I guess here we only have target to ignore

Reply:
good call